src/_components/ForStudents/ApplicationProcess.jsx

```diff
@@ -0,0 +1,165 @@
+import React from "https://esm.sh/react";
+
+export default function ApplicationProcess({ comp }) {
+  return (
+    <section className="flex flex-col items-center px-12 pb-12 max-lg:px-12 max-md:w-full max-md:px-4">
```

Comment:
`nit`: The top padding of the application changes does not correspond. to the paddings elsewhere in the `/students` page.
<img width="1624" alt="image" src="https://github.com/user-attachments/assets/3a3e421b-9065-444f-8da1-f4d734439e33">
<img width="1624" alt="image" src="https://github.com/user-attachments/assets/570c21e5-cf81-45ed-a99d-560cf8faf9b7">
Try to fix the padding, so that they are both the same. You will need to change the top padding in this component.

Reply:
Yes please

Reply:
I pushed a commit fixing the bottom padding of the abovementioned component, though the extra space can also be attributed to its weird height. I talked to Christian yesterday and it should be okay for now, just wanted to check if the spacing is also okay for you now.